### ISSUE TITLE
Folder contents view: Introduce "View" button in toolbar

### ISSUE THREAD

**Issue by ksuess** [plone/volto#3713]
I suggest to rename the "back" button to "view". And choose an icon accordingly.

![grafik](https://user-images.githubusercontent.com/1144737/193471334-93275db1-b0c0-45ef-8b34-1c257d0704b8.png)

**Comment by tiberiuichim** [comment]
I agree with the idea, but maybe we need another icon? The problem I see is that, if you flip the arrow 180, you get our "ok" icon. Or, maybe, that Ok icon, where it's used, should be replaced by the Save icon (floppy disk) ?

**Comment by stevepiercy** [comment]
For me, "view" does not tell me what I will view. "view contents" would tell me that.

Similarly, the icon's alt/title/hover text that lists contents is just "Contents", instead of "browse contents" or "list contents".

![Screen Shot 2022-10-03 at 12 21 09 AM](https://user-images.githubusercontent.com/102112/193521515-04f25cae-e7b0-4a51-b355-8fb8f7716b1c.png)

I would suggest `content-listing.svg` and `content-existing.svg` as the pair of icons to use.

https://pastanaga.io/icons/

Whatever is decided, the two icons and alt/title/hover text should align with their respective icon and make sense as a complementary pair of actions/icons.

**Comment by ksuess** [comment]
My argument for replacing the "back" button in contents view is, that it is in fact not a back. Imagine a user navigating in contents view, found what he was looking for and than wants to view the page. So I think it's more a "view" action. 

Maybe the show.svg could be a candidate.
![grafik](https://user-images.githubusercontent.com/1144737/193525799-38fdd610-13e3-4a64-9d9e-28ea14637f4e.png)

What do you think?

**Comment by ksuess** [comment]
OK. Maybe if me or someone else who is interested, has some minutes left over, we could make the buttons overridable by retrieving them optionally from the component registry.

**Comment by ksuess** [comment]
Not planned. Closing for next editor theme.
